vm-systemd/qubes-network-uplink@.service

```diff
@@ -3,6 +3,8 @@ Description=Qubes network uplink (%i) setup
 After=network-pre.target qubes-iptables.service
 After=sys-subsystem-net-devices-%i.device
 BindsTo=sys-subsystem-net-devices-%i.device
+After=systemd-sysctl.service
+BindsTo=systemd-sysctl.service
```

Comment:
I don't think `BindsTo` is correct here, stopping interfaces together with systemd-sysctl doesn't make sense.

Reply:
I will use `Requires` instead.

Reply:
I wound up omitting it since all of the hardening is also done in the script itself.